package.json

```diff
@@ -17,12 +20,14 @@
     },
     "main": "./out/src/vscgettext",
     "engines": {
-        "vscode": "^1.12.0"
+        "vscode": "^1.70.0"
```

Comment:
This sadly breaks the extension for old vscode installations.
I totally agree that 1.12 is _very_ outdated, but @QuentiumYT is 1.70 really needed for your changes? [yes, I still run 1.5x versions in places that aren't easily upgradable]

Reply:
Thanks for the advice! I can perform type fix in #27 🔧

Reply:
Yeah you're correct. It's a huge bump and I didn't think about old VSCode users that cannot easily update. It can be changed back to 1.50 :) sorry for the inconvenience

Reply:
Thanks for checking and for taking care in the first place!